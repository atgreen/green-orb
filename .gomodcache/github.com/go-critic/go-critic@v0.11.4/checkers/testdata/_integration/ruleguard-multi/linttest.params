check -@ruleguard.rules rules1.go,rules2.go -enable ruleguard ./... | linttest.golden