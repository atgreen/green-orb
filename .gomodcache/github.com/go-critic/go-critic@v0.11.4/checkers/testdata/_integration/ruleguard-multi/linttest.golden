exit status 1
./file.go:20:2: ruleguard: maybe mu.RLock() was intended?
./file.go:10:16: ruleguard: suggestion: filepath.Separator
